### PR TITLE
migrated all geo functionality to work with  PostGIS 2.x

### PR DESCRIPTION
**Pull request by voisardf**
Adapted the code of core to work with postgis 2.x. @kalbermattenm : Please review when you have some spare time =;0) thanks

### CODE REVIEW

File: crdppf/util/pdf_functions.py [hunk 1]
```diff
@@ -6,7 +6,9 @@
 # geometry related librabries
 from shapely.geometry import Point as splPoint, Polygon as splPolygon
 from shapely.geometry import MultiPolygon as splMultiPolygon, LinearRing as splLinearRing
-from geoalchemy import WKTSpatialElement
+from geoalchemy2 import functions, WKTElement, WKBElement, shape
```
Comment (kalbermattenm) [inline crdppf/util/pdf_functions.py]: I do not think that you use `WKBElement` and `shape` in this code file ? If not, please remove from the imports...

**Comment by kalbermattenm** [comment]
Just a comment, please address and merge.

**Comment by kalbermattenm** [comment]
Wait... DO NOT MERGE... Just address the comment...

But please DO NOT MERGE !!!!!

File: crdppf/util/pdf_functions.py (hunk 1 above)
Comment (voisardf) [inline crdppf/util/pdf_functions.py]: you got me there - already on it =:0)

**Comment by voisardf** [comment]
no worries! I'm not gona merge it... YET

**Pull request merged**